plugins/grpc-ecosystem/gateway/v2.20.0/separate_pkg_additional_imports.patch

```diff
@@ -237,7 +237,9 @@ index 62766254..8084ff74 100644
 +		//
 +		// If gateway has a new major version, we should then stop generating at the wrong path.
 +		aliasedPackage := &descriptor.GoPackage{
-+			Path:  goPkg.Path,
++			// When generating for generated SDK, the original goPkg points to code generated by "protocolbuffers/go",
++			// but we are aliasing to a package generated by "grpc-ecosystem/gateway".
++			Path:  strings.Replace(goPkg.Path, "protocolbuffers/go", "grpc-ecosystem/gateway", 1),
```

Comment:
I think we already have constants for protocolbuffers/go and grpc/go in internal/descriptor/buf_build.go. It would be good to centralize all of those in one place.

We should also consider changing the replace to be more specific `/protocolbuffers/go` -> `/grpc-ecosystem/gateway`.

Reply:
Fixed in [f8c1369](https://github.com/bufbuild/plugins/pull/1379/commits/f8c1369107878d8cff05d84d24d451b5e634feba) and [868f508](https://github.com/bufbuild/plugins/pull/1379/commits/868f50836f39c185b2f9a6c97a7a3107bb4c9f84)